htdocs/expedition/class/expedition.class.php

```diff
@@ -2257,6 +2256,138 @@ function reOpen()
 		}
 	}
 
+	// phpcs:disable PEAR.NamingConventions.ValidFunctionName.NotCamelCaps
```

Comment:
phpcs rule was introduced to avoid adding non camel caps function

Reply:
Hello, yes i understand but i wanted to keep the format of this function like in other object see : https://github.com/Dolibarr/dolibarr/blob/8ee98315c750b2fdd24c055dc6a1cd2d7a1ffee4/htdocs/commande/class/commande.class.php#L461

Reply:
so don't forget the phpcs:enable

Reply:
https://github.com/Dolibarr/dolibarr/commit/c0b6699b253dbcc8aeba51bc90e3d4f1a9323e32
all methods have been renamed to setDraft